src/Virtphp/Workers/Activator.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+/**
+ * This file is part of virtPHP.
+ *
+ * (c) Jordan Kasper <github @jakerella>
+ *     Ben Ramsey <github @ramsey>
+ *     Jacques Woodcock <github @jwoodcock>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Virtphp\Workers;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\TableHelper;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Activator extends AbstractWorker
+{
+
+    /**
+     * @var string
+     */
+    protected $envName;
+
+    /**
+     * @var string
+     */
+    protected $envPath;
+
+    /**
+     * @var string
+     */
+    protected $envFile;
+
+    /**
+     * @var string
+     */
+    protected $file = 'environments.json';
```

Comment:
Should this be `protected` or a constant? Can it ever change during execution? (Or should it?)